src/Uno.UI/UI/Xaml/Controls/NavigationView/NavigationView.cs

```diff
@@ -83,7 +83,7 @@ private void UpdateMinimalMode()
 
 			if (_togglePaneButton != null && _navigationViewBackButton != null)
 			{
-				_togglePaneButton.Margin = new Thickness(0, _navigationViewBackButton.RenderSize.Height, 0, 0);
+				_togglePaneButton.Margin = new Thickness(0, _navigationViewBackButton.Height, 0, 0);
```

Comment:
why this change? What's going to happen if Height isn't set? I don't even know what a Thickness with NaN in it does.

Reply:
RenderSize is not updated properly when calling this method, We could problably fail if the size if not properly set, as a last resource. This should have been done as a visual state, but that's not how the default style works...